eox_tenant/edxapp_wrapper/backends/enrollments_h_v1.py

```diff
@@ -0,0 +1,11 @@
+""" backend """
+from common.djangoapps.student import models  # pylint: disable=import-error
```

Comment:
```suggestion
from student import models  # pylint: disable=import-error
```
This solves the import problem.